integrationtests/wildfly-modules/src/test/java/org/infinispan/test/integration/as/InfinispanStoreRocksDBIT.java

```diff
@@ -55,9 +69,12 @@ public void testCacheManager() {
       GlobalConfigurationBuilder gcb = new GlobalConfigurationBuilder();
       gcb.globalJmxStatistics().allowDuplicateDomains(true);
 
+      String tmpDirectory = tmpDirectory(this.getClass());
       ConfigurationBuilder builder = new ConfigurationBuilder();
-      builder.persistence().addStore(RocksDBStoreConfigurationBuilder.class)
-            .location(tmpDirectory(this.getClass()));
+      builder.persistence()
+             .addStore(RocksDBStoreConfigurationBuilder.class)
+             .location(tmpDirectory)
+             .expiredLocation(tmpDirectory + "-expired");
```

Comment:
I think RocksDB needs both those directories (but I might be wrong). Maybe @tristantarrant would know.